src/containers/BooksList/BooksList.js

```diff
@@ -1,15 +1,22 @@
-import { useState } from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
- [ ] It would be better if we address the comment mentioned by the previous code reviewer. I would highlight the requirements so they can be addressed. In the BookList component, we need to I. implement handleRemoveBook ii. connect removeBook action iii. pass handleRemoveBook to the Book component. Here is a screenshot of the requirements
![Screenshot from 2021-09-01 15-54-05](https://user-images.githubusercontent.com/46388113/131704197-1da388b7-ea7d-4d13-b118-6170c22bda26.png)

Reply:
Actually in this project I have used hooks which cannot be used in nested function, the method which is mentioned uses mapDispatchtoProp with connect function, here action is being connected with hooks "useDispatch" which works same as mapDispatchtoProp and connect 
Its nowhere mentioned that I cannot use hooks to do the same, I wanted to do with functional component hence I used hook and it is same way functioning as it should with mapDispatchtoProp and connect 

![Screenshot from 2021-09-01 23-43-26](https://user-images.githubusercontent.com/73999611/131722481-69384186-a38f-4417-a012-78551935824d.png)